Carbon/util/SustainabilityScoreProfileView.js

```diff
@@ -0,0 +1,48 @@
+//Titles for the SustainabilityScores - Server only stores the ints, Client side will translate them to the titles and path into the picture,
+//  Adding A's and An's to avoid future formatting annoyances
+//  NOTE: users MUST provide their own relative path to the Carbon directory
+export const SustainabilityScoreProfileView = [
+    {
+        title : "An Eco Opponent",
+        picture : "assets/rank-icons/rank-icon-1.png"
+    },
+    {
+        title : "A Climate Change Denier",
+        picture : "assets/rank-icons/rank-icon-1.png"
+    },
+    {
+        title : "A Sustainability Saboteur",
+        picture : "assets/rank-icons/rank-icon-2.png"
+    },
+    {
+        title : "A Planet Polluter",
```

Comment:
I hate to be a pain, but can we get rid of 'A' and 'An' from these names. I think it makes more sense to just say the title.

Reply:
Sounds good, just removed them. I'm marking this as resolved. 👍